### PR TITLE
Abh upstream v1.10.3

### PR DESCRIPTION
**Pull request by ABaldwinHunter**
Update our fork of eslint.

Note: eslint made some changes in the configuration that intersected with our changes to permit airbnb packages only. We updated our code accordingly.

cc @codeclimate/review

### CODE REVIEW

**Comment by ABaldwinHunter** [comment]
This change locks our fork to `v1.10.3` of eslint.

New commit to permit only airbnb config package is https://github.com/codeclimate/eslint/commit/2af95dc45bf177dbd3188dde84505cef64c8e71f

File: lib/config/config-file.js [hunk 1]
```diff
@@ -0,0 +1,444 @@
...
+ * @throws {Error} If the package cannot be read.
+ * @private
+ */
+function loadPackage(filePath) {
+    debug("Loading config package: " + filePath);
+    try {
+        if (filePath.match(/^eslint-config-airbnb.*/)) {
+            return require(filePath);
+        } else {
+            return {}
```
Comment (gdiggs) [inline lib/config/config-file.js]: :lipstick: Missing semicolon

**Comment by gdiggs** [comment]
Minor style thing but otherwise LGTM

**Comment by ABaldwinHunter** [comment]
thanks @GordonDiggs. Fixed :+1:

**Comment by gdiggs** [comment]
:+1:

**Pull request merged**